tracepoint_decode/src/perf_item.rs

```diff
@@ -85,7 +85,7 @@ impl PerfConvertOptions {
     }
 
     /// Returns true if `self & flag != 0`.
-    pub const fn has(self, flag: Self) -> bool {
+    pub const fn has_flag(self, flag: Self) -> bool {
```

Comment:
Note: Renamed for consistency - several other enums in this project are using `has_flag`.